Example/Controllers/SwiftUI/Features/Settings/Features/MiniAppBridgeView.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import MiniApp
+
+struct MiniAppBridgeView: View {
```

Comment:
MiniAppBridgeView is too generic that it could mean the entire bridge used to communicate MiniApp. May be "MessageBridgeView" or "UniversalBridgeView"?

Reply:
Sure got the point, I have updated to use the name `UniversalBridgeView` also update the relevant class file name.